yaml/src/main/java/module-info.java

```diff
@@ -11,7 +11,8 @@
     exports tools.jackson.dataformat.yaml.util;
 
     provides tools.jackson.core.TokenStreamFactory with
-        tools.jackson.dataformat.yaml.YAMLFactory;
+        tools.jackson.dataformat.yaml.YAMLFactory,
+        tools.jackson.dataformat.yaml.YAMLAnchorReplayingFactory;
```

Comment:
I think we should not expose this via SPI but only standard factory.

Reply:
Ok, to be fair, I don't know very well how modules work and what does this `provides` do. I just saw it was there and thought maybe it was needed for something haha.
Removed it :+1: